src/storm-pars/modelchecker/region/ValidatingSparseMdpParameterLiftingModelChecker.cpp

```diff
@@ -16,7 +16,7 @@ namespace storm {
 
 
         template <typename SparseModelType, typename ImpreciseType, typename PreciseType>
-        void ValidatingSparseMdpParameterLiftingModelChecker<SparseModelType, ImpreciseType, PreciseType>::specify(Environment const& env, std::shared_ptr<storm::models::ModelBase> parametricModel, CheckTask<storm::logic::Formula, typename SparseModelType::ValueType> const& checkTask, bool generateRegionSplitEstimates, bool allowModelSimplifications) {
+        void ValidatingSparseMdpParameterLiftingModelChecker<SparseModelType, ImpreciseType, PreciseType>::specify(Environment const& env, std::shared_ptr<storm::models::ModelBase> parametricModel, CheckTask<storm::logic::Formula, typename SparseModelType::ValueType> const& checkTask, [[maybe_unused]] bool generateRegionSplitEstimates, [[maybe_unused]] bool allowModelSimplifications) {
```

Comment:
I would assume that these should not be unused? @jipspel @tquatmann

Reply:
Yes, here we should at least forward the flag to the two  `specify` calls of the underlying `(im)`preciseChecker`s in lines 31 and 32.